### PR TITLE
Reset flags for file loggers

### PR DESCRIPTION
**Pull request by ataymano**
By default logs got appended to previous ones, reset=True is starting files from scratch

### CODE REVIEW

File: from_mwt_ds/DataScience/vw_executor/loggers.py [hunk 1]
```diff
@@ -109,16 +109,25 @@ class FileLogger(ILogger):
...
+            path.parent.mkdir(exist_ok=True, parents=True)
+            impl = _FileLoggerSafe(path)
         super().__init__(impl, logging.getLevelName(self.level_str), tag)
 
     def __getitem__(self, key: str) -> 'FileLogger':
-        return FileLogger(path=None, level=self.level_str, tag = f'{self.tag}[{key}]', impl=self.impl)
+        return FileLogger(
+            path=None,
+            level=self.level_str,
+            reset=False,
```
Comment (lalo) [inline from_mwt_ds/DataScience/vw_executor/loggers.py:128]: shouldn't this also be self.reset?

File: from_mwt_ds/DataScience/vw_executor/loggers.py [hunk 2]
```diff
@@ -109,16 +109,25 @@ class FileLogger(ILogger):
     def __init__(self,
                  path: Optional[Union[str, Path]],
                  level: str = 'INFO',
+                 reset: bool = False,
```
Comment (lalo) [inline from_mwt_ds/DataScience/vw_executor/loggers.py:112]: maybe adding docstring would be good, to mention that is to rewrite the file and not other behavior regarding the scoping/sub logger functionality.

File: from_mwt_ds/DataScience/vw_executor/loggers.py (hunk 1 above)
Comment (ataymano) [reply from_mwt_ds/DataScience/vw_executor/loggers.py:128]: we are not using it anywhere other than in constructor

File: from_mwt_ds/DataScience/vw_executor/loggers.py (hunk 2 above)
Comment (ataymano) [reply from_mwt_ds/DataScience/vw_executor/loggers.py:112]: yeah, let me add it.

**Review by lalo — APPROVED** [review]

**Pull request merged**